packages/ckeditor5-ckbox/docs/features/ckbox.md

```diff
@@ -83,18 +83,20 @@ These plugins are, by default, loaded with predefined builds, such as `ClassicEd
 
 Please also remember, that the CKBox plugin requires the following dependency plugins to work properly: `ArticlePluginSet`, `PictureEditing`, `ImageUpload`, and `CloudServices`. Except for `ImageUpload` which is, likewise, available in predefined builds, these need to be added manually.
 
-Finally, add {@link module:ckbox/ckbox~CKBox} to your plugin list and [configure](#configuration) the feature as needed. An example configuration may look like this:
+If you want to be able to use CKBox image editing capabilities from within CKEditor&nbsp;5, you need to also include the `CKBoxImageEdit` plugin.
```

Comment:
Maybe I would reverse the order of this sentence to something like:
> You must include the `CKBoxImageEdit` plugin if you want to use CKBox image editing capabilities from within CKEditor 5.

Reply:
In commit.